src/client/driver/driver.js

```diff
@@ -275,12 +276,25 @@ export default class Driver extends serviceUtils.EventEmitter {
     }
 
     // Error handling
-    _onJsError (err) {
+    async _onJsError (err) {
         // NOTE: we should not send any message to the server if we've
         // sent the 'test-done' message but haven't got the response.
-        if (this.skipJsErrors || this.contextStorage.getItem(TEST_DONE_SENT_FLAG))
+        if (this.contextStorage.getItem(TEST_DONE_SENT_FLAG))
```

Comment:
The `_onJsError (err)` has strange return value signature in old version - `Promise` or `null`.
In updated version you always returns `Promise`.
I guess we just use the `return` keyword. Please check it.